projects/DiffusionDet/README.md

```diff
@@ -60,14 +60,30 @@ We give a table to compare the inference results on `ResNet50-500-proposals` bet
 
 ### Training commands
 
-MMDetection currently does not fully support training DiffusionDet.
+In MMDetection's root directory, run the following command to train the model:
+
+```bash
+python tools/train.py projects/DiffusionDet/configs/diffusiondet_r50_fpn_500-proposals_1-step_crop-ms-480-800-450k_coco.py
+```
+
+For multi-gpu training, run:
+
+```bash
+python -m torch.distributed.launch --nnodes=1 --node_rank=0 --nproc_per_node=${NUM_GPUS} --master_port=29506 --master_addr="127.0.0.1" tools/train.py projects/DiffusionDet/configs/diffusiondet_r50_fpn_500-proposals_1-step_crop-ms-480-800-450k_coco.py
+```
 
 ### Testing commands
 
 In MMDetection's root directory, run the following command to test the model:
 
 ```bash
-python tools/test.py projects/DiffusionDet/configs/${CONFIG_PATH} ${CHECKPOINT_PATH}
+# for 1 step inference
+export CONFIG_FILE=projects/DiffusionDet/configs/diffusiondet_r50_fpn_500-proposals_1-step_crop-ms-480-800-450k_coco.py
```

Comment:
Can we use `--cfg-options model.bbox_head.sampling_timesteps=4` to control steps? It's much simpler.

Reply:
So we can delete 4 step config file?

Reply:
> So we can delete 4 step config file?

I agree with deleting 4 step config!